src/app/client/src/app/modules/workspace/components/collaboration-content-filter/collaboration-content-filter.component.html

```diff
@@ -4,10 +4,10 @@
         <div class="searchfilterinput ui search">
           <div class="searchicon ui icon input ">
             <input class="upForReviewSearchBox" type="text" name="filter_search" 
-              [(ngModel)]="query" (keyup)="keyup(query)"
+              [(ngModel)]="query" (keyup)="keyup(query)" appTelemetryInteract [telemetryInteractEdata]="setCollabartionContentSearchInteractEdata()"
```

Comment:
this is not right, we need telemetry to be generated only on api call.

Reply:
InteractEvent is for only on click not on auto search or timeout.
appTelemetryInteract this directive for only on click